lighthouse-cli/test/smokehouse/pwa-expectations.js

```diff
@@ -0,0 +1,128 @@
+    url: 'https://airhorner.com/',
+    audits: {
+      'is-on-https': true,
+      'redirects-http': true,
+      'service-worker': true,
+      'works-offline': true,
+      'manifest-display': true,
+      'manifest-exists': true,
+      'manifest-background-color': true,
+      'manifest-theme-color': true,
+      'manifest-icons-min-192': true,
+      'manifest-icons-min-144': true,
+      'manifest-name': true,
+      'manifest-short-name': true,
+      'manifest-start-url': true,
+      // 'cache-start-url': true
+    }
+  },
+
+  {
+    initialUrl: 'https://www.chromestatus.com/',
+    url: 'https://www.chromestatus.com/features',
+    audits: {
+      'is-on-https': true,
+      'redirects-http': true,
+      'service-worker': true,
+      'works-offline': false,
+      'manifest-display': true,
+      'manifest-exists': true,
+      'manifest-background-color': true,
+      'manifest-theme-color': true,
+      'manifest-icons-min-192': true,
+      'manifest-icons-min-144': true,
+      'manifest-name': true,
+      'manifest-short-name': true,
+      'manifest-start-url': true,
+      // 'cache-start-url': true
+    }
+  },
+
+  {
+    initialUrl: 'https://jakearchibald.github.io/svgomg/',
+    url: 'https://jakearchibald.github.io/svgomg/',
+    audits: {
+      'is-on-https': true,
+      'redirects-http': true,
+      'service-worker': true,
+      'works-offline': true,
+      'manifest-display': true,
+      'manifest-exists': true,
+      'manifest-background-color': true,
+      'manifest-theme-color': true,
+      'manifest-icons-min-192': true,
+      'manifest-icons-min-144': true,
+      'manifest-name': true,
+      'manifest-short-name': true,
+      'manifest-start-url': true,
+      // 'cache-start-url': true
+    }
+  },
+
+  {
+    initialUrl: 'https://shop.polymer-project.org/',
+    url: 'https://shop.polymer-project.org/',
+    audits: {
+      'is-on-https': true,
+      'redirects-http': true,
+      'service-worker': true,
+      'works-offline': true,
+      'manifest-display': true,
+      'manifest-exists': true,
+      'manifest-background-color': true,
+      'manifest-theme-color': true,
+      'manifest-icons-min-192': true,
+      'manifest-icons-min-144': true,
+      'manifest-name': true,
+      'manifest-short-name': true,
+      'manifest-start-url': true,
+      // 'cache-start-url': true
+    }
+  },
+
+  {
+    initialUrl: 'https://pwa.rocks',
+    url: 'https://pwa.rocks/',
+    audits: {
+      'is-on-https': true,
+      'redirects-http': true,
+      'service-worker': true,
+      'works-offline': true,
+      'manifest-display': true,
+      'manifest-exists': true,
+      'manifest-background-color': true,
+      'manifest-theme-color': true,
+      'manifest-icons-min-192': true,
+      'manifest-icons-min-144': true,
+      'manifest-name': true,
+      'manifest-short-name': true,
+      'manifest-start-url': true,
+      // 'cache-start-url': true
+    }
+  }
+];
```

Comment:
> Some fairly static PWAs are listed here where we know the sources may be unlikely to majorly change (with the exception of ChromeStatus). Was this intentional?  

It was. The idea is that this could graduate to something that runs on travis. The corpus needs to be fairly stable, where we aren't constantly tracking down failures to see if it was something on the site that changed or if it was us, so demo PWAs seem like a good pick. Not sure yet where this will live in the testing pipeline (every PR? daily cron job? dunno), so how long this should take to run (number of sites in the test and the number of slow sites) is also up in the air. It's possible this is fast enough to add to the normal travis run, and then we have a more extensive one on a daily run and which blocks release.

Definitely just a working first effort for this PR, but also happy to add more/different test sites.